schemas/dto/recordProcessingLogDto.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Record processing log DTO schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "jobExecutionId": {
+      "description": "UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "sourceRecordId": {
+      "description": "SRS record identifier, UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "sourceRecordOrder": {
+      "description": "Order of a record in imported file",
+      "type": "integer",
+      "minimum": 0
```

Comment:
Is it possible to set the minimum to `1`? Because the order of the first record in the table is `1`.

Reply:
To be honest, actually, a source record number starts with 0 and it is stored in a such way on the back-end side. So, it would be good if we could increment it on the ui side. Could you let me know if it's possible? Otherwise, we will do some workaround on the back-end side.

Reply:
Ok, no problem, we can increment it on the UI

Reply:
Thank you @mariia-aloshyna!